code/src/components/LessonRadioOption.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const lessons = [
+    'Yeah, big time',
+    'Some things',
+    'Not really'
+]
+ 
+const LessonRadioOption = ({ setLesson }) => {
+    const onLessonChanged = (e) => {
+        setLesson(e.target.value)
+    }
+ 
+    return (
+        <section className="page-container">
+            <header className="header-container-radio">
+                <h1>Has the quarantine lifestyle made you reconsider what's important in your life?</h1>
+            </header>
+            {lessons.map((lesson) => {
```

Comment:
Good idea to make a .map instead of manually inputting the attributes for each option, as it makes it easy to add more options in the future (future-proofing is maybe outside of assignment scope, but good practice for larger apps in the future!)